app/helpers/news_helper.rb

```diff
@@ -1,17 +1,21 @@
 module NewsHelper
-  def news_article_class
+  def news_articles#(section: nil)
```

Comment:
This will conflict with the changes I have made in the other PR. Just a heads up :)